src/api/registration/registration.controller.js

```diff
@@ -61,6 +59,24 @@ const createNewRegistration = async (
     hygieneCouncilCode
   );
 
+  const completeCacheRecord = Object.assign(
+    {},
+    {
+      "fsa-rn": postRegistrationMetadata["fsa-rn"],
+      reg_submission_date: postRegistrationMetadata.reg_submission_date
+    },
+    registration,
+    lcContactConfig,
+    {
+      status: {
+        registration: undefined,
```

Comment:
Since, we're only using true / false now, we can initialise with false and not have to update on failure